code_generator/Improbable.TextTemplating/Improbable.TextTemplating.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can we import the nuget package? https://www.nuget.org/packages/Mono.TextTemplating

Reply:
We unfortunately can't because the last version there is 1.5 years old and doesn't support .NET Core 2